doc/jupyter_intro.md

```diff
@@ -0,0 +1,261 @@
+                                 flags="b",
+                                 border_color="none")
+elevation_3dmap.show()
+```
+
+![Landuse map draped over elevation with gj.Map3D](jupyter_3d_map.png)
+
+The parameters of the `render()` function are the same as parameters of the
+[m.nviz.image](m.nviz.image.md) tool, which is used in the background.
+
+## Series map
+
+The `gj.SeriesMap` class animates a series of maps, allowing users to slide between
+maps and play a continuous loop.
+
+```python
+# Create a series of relief maps with different angles
+directions = [0, 90, 180, 270]
+for azimuth in directions:
+    gs.run_command("r.relief",
+                   input="elevation",
+                   output=f"relief_{azimuth}",
+                   azimuth=azimuth)
+m = gj.SeriesMap()
+m.add_rasters(f"relief_{azimuth}" for azimuth in directions)
+m.d_vect(map="roads")
+m.d_barscale(at=(80, 10))
+m.show()
+# Save the map as animated gif
+m.save("series_map.gif")
+```
+
+![Changing relief direction example with gj.SeriesMap](jupyter_series_map.png)
+
+## Time series map
+
+The `gj.TimeSeriesMap` class provides a way to visualize
+GRASS' [space time datasets](temporalintro.md) in Jupyter. Here we create a time
+series map of overland water flow:
+
+```python
+# Zoom in to the study area
+gs.run_command("g.region", n=226040, s=223780, e=639170, w=636190)
+# Compute topography dx and dy derivatives
+gs.run_command("r.slope.aspect",
+               elevation="elevation",
+               dx="dx",
+               dy="dy")
+# Compute overland flow
+gs.run_command("r.sim.water",
+               flags="t",
+               elevation="elevation",
+               dx="dx",
+               dy="dy",
+               depth="depth",
+               niterations=30)
+
+# Create a time series
+gs.run_command("t.create",
+               output="depth",
+               temporaltype="relative",
+               title="Overland flow depth",
+               description="Overland flow depth")
+# Register the time series
+maps = gs.list_strings(type="raster", pattern="depth*")
+gs.run_command("t.register", input="depth", maps=maps)
+
+# Create a time series map
+flow_map = gj.TimeSeriesMap()
+
+# Add the base map
+flow_map.d_rast(map="elevation")
+# Add the time series data
+flow_map.add_raster_series("depth", values="0.025-1.25")
+# Overlay the vector map
+flow_map.d_vect(map="streets")
+
+# Add map features
+flow_map.d_legend(raster="depth.30",
+                  title="Depth [m]",
+                  flags="bt",
+                  border_color="none",
+                  at=(1, 50, 0, 5))
+
+# Display the map
+flow_map.show()
+```
+
+![Water flow example with gj.TimeSeriesMap](jupyter_timeseries_map.png)
+
+## Python library documentation
+
+For complete documentation on the `grass.jupyter` package, see the
+[grass.jupyter](https://grass.osgeo.org/grass-stable/manuals/libpython/grass.jupyter.html)
+library documentation page.
```

Comment:
grass.script or general grass packages would be good to include. You have grass.script used, but the link is missing. The heading is "Python library documentation". It could also link to the Python "scripting" intro.